### ISSUE TITLE
[A11y] improve basket quantity cell for screen reader

### ISSUE THREAD

**Issue by wiffbi** [pretix/pretix#2213]
**Describe the bug**
When usin g a screen reader it is hard to get the quantity of an item in the basket as the quantity cell starts with the remove button.

**Expected behavior**
When focussing the quantity cell, the quantity should be read first. Either put quantity first in source-order and use flexbox in quantity-cell to reorder items visually. Or add visually-hidden quantity and aria-hidden the visual quantity.
